### DISCUSSION TITLE
History: users are asking how to remove products from history

### DISCUSSION THREAD

**Discussion by CharlesNepote** [openfoodfacts/openfoodfacts-androidapp#4023] [Q&A]
Some users are asking how to remove some products (not all) from the history list.

In fact there is no way to guess that we must select the desired item, and get the finger from right to left.

I have no idea how to deal with that, but we had two questions by email recently (email at contact@openfoodfacts.org on 2021-03-31, 2021-03-28).

**Comment by teolemon** [comment]
We don't have any onboarding for this gesture.
